pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -197,6 +198,7 @@ func BuildKubernetesMasterConfig(options configapi.MasterConfig, requestContextM
 			PublicAddress: publicAddress,
 			ReadWritePort: port,
 
+			Authenticator:    originAuthenticator, // this is used to fulfill the tokenreviews endpoint which is used by node authentication
```

Comment:
does this cause the authenticator to be invoked multiple times for k8s API requests?